src/metadata.rs

```diff
@@ -167,9 +173,11 @@ fn test_read_metadata() {
 		artist: Some("TEST ARTIST".into()),
 		album_artist: Some("TEST ALBUM ARTIST".into()),
 		album: Some("TEST ALBUM".into()),
+        duration: None,
 		year: Some(2016),
 	};
+	let flac_sample_tag = SongTags {duration: Some(0), ..sample_tags.clone()};
```

Comment:
It might be worth replacing the sample Flac file with something a bit more meaningful to test duration (as long as it's copyright-free).

Reply:
yeah i thought about that, because the test files have duration of zero seconds, but with your  update, the default when there's no duration becomes `None`